channeldb/invoice_test.go

```diff
@@ -675,6 +673,7 @@ func TestQueryInvoices(t *testing.T) {
 	if err != nil {
 		t.Fatalf("unable to make test db: %v", err)
 	}
+	db.Now = func() time.Time { return testNow }
```

Comment:
nit: To make it more explicit that the same now function is now used everywhere we could extract this function declaration to the top as well and maybe call it `testNowFn`.

Reply:
Probably best fix is to add `Clock` to the database.